NewsFeed/views/CCBRNewsViewController.m

```diff
@@ -85,7 +99,27 @@ - (void)viewDidLoad {
 - (void)updateUI {
     self.collectionView.hidden = self.viewModel.collectionViewHidden;
     self.errorMessageLabel.hidden = self.viewModel.errorMessageLabelHidden;
-    [self.collectionView reloadData];
+    self.loadingIndicatorView.hidden = !self.viewModel.indicatorViewLoading;
+    if (self.viewModel.indicatorViewLoading) {
+        [self.loadingIndicatorView startAnimating];
+    } else {
+        [self.loadingIndicatorView stopAnimating];
+    }
```

Comment:
If we set the `hidden` property of the `loadingIndicatorView`, then we don't have to call `startAnimating` or `stopAnimating` anymore.